
uint32 id
Vec2DFlt32 start
Vec2DFlt32 end
